src/comp/comp.py

```diff
@@ -23,49 +23,97 @@ def __repr__(self):
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
+print('\n')
+
 print("Starts with D:")
 a = []
+
+for i in humans:
+    if 'D' in i.name[0]:
+        a.append(i.name)
```

Comment:
These are for loop. We need to use list comprehensions for solving all the problems in this sheet. Like this:

a = [human for human in humans]